movies-manager/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "movies-manager",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build:dev": "cross-env NODE_ENV=development webpack-dev-server",
```

Comment:
Why you have no 'cross-env' package in dependencies?

Reply:
Please find out and write here which scripts should have react app (or for what reasons all this 4 scripts). I can see that you have no understanding (build.dev for example).

Reply:
Also read task 2 one again.

Reply:
> Why you have no 'cross-env' package in dependencies?

done

Reply:
> Also read task 2 one again.

done